src/rgb/standards.rs

```diff
@@ -0,0 +1,53 @@
+//!Various RGB standards.
+
+use num::Float;
+use Yxy;
+use flt;
+use white_point::{WhitePoint, D65};
+use rgb::{Primaries, RgbSpace, RgbStandard};
+use pixel::TransferFn;
+
+///The sRGB color space.
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub struct Srgb;
+
+impl Primaries for Srgb {
```

Comment:
Look at http://ninedegreesbelow.com/photography/srgb-color-space-to-profile.html#sRGB-profile-source-destination-white-points Table 6 and Table 7. The XYZ values are different.

Without the white point in the primaries, we don't know if chromatic adaptation is required or not for the primaries. Example ```Rgb<Srgb, Srgb, D65>```, no adaptation required since white point and primaries (white point) match, while ```Rgb<Srgb, Srgb, D50>``` will require adaptation.